frontend/src/components/Header.jsx

```diff
@@ -1,24 +1,34 @@
 import { Link } from "react-router-dom"
+import { useEffect, useState } from "react"
 
 import TextSearch from "./TextSearch"
 import gatsbyLogo from "../images/GATSBY.jpeg"
 import "../style/Header.css"
 
 export default function Header({ searchHandler }) {
+  const [currentPage, setCurrentPage] = useState()
```

Comment:
Know this isn't ts. But maybe initialise this state with a value that lets future devs know what to expect. In this case an empty string? Whatcha think?

Reply:
yeah, I'm actually doing it now for the other state as well that just needs to be an array